### DISCUSSION TITLE
snow

### DISCUSSION THREAD

**Discussion by SupaDupa24** [matteobruni/tsparticles#1164] [General]
quick question has anyone been able to get snow falling with either tsparticles or react particles?

**Comment by adamalston** [comment]
There is a demo of snow config available at [https://rpj.bembi.dev/#snow](https://rpj.bembi.dev/#snow).

**Comment by SupaDupa24** [comment]
oh yeah the example he gave me worked perfectly just had to adjust to
canvas of project with some css
